src/main/apps/sourceChanges.ts

```diff
@@ -34,11 +34,6 @@ const downloadSource = async (
     const sourceJson = await downloadSourceJson(url);
     const source: Source = { name: sourceJson.name, url };
 
-    const isLegacyUrl = url.endsWith('/apps.json') && sourceJson.apps == null;
-    if (isLegacyUrl) {
-        return downloadSource(url.replace(/apps\.json$/, 'source.json'));
-    }
-
```

Comment:
Just to make sure I get the intention here, but why is this removed?

Reply:
Almost two years after we switched from using `apps.json` to `source.json` files, I consider this code to be unneeded and wanted to clean up by removing it. 

The only case when it would still be used, would be if someone would add a URL like https://developer.nordicsemi.com/.pc-tools/nrfconnect-apps/test/apps.json (instead of …/source.json) in the dialog to add a source.